Index.html

```diff
@@ -9,6 +9,13 @@
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Inter">
 </head>
 <body>
+    <header>
```

Comment:
- [ ] There is soo much blank space at the top of your header, kindly remove that please to match the figma template. You are expected to stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
### Your design
![3](https://user-images.githubusercontent.com/94551484/206829917-b14d09cd-9be2-4bca-a53b-24ad8a29dcc9.png)

Reply:
Hello @Kamba56, 
Thanks for the review. The space at the top of the mobile version is actually part of the figma template. In it were a battery icon, time, and network signal indicators which I was told I should not include in previous reviews. However, I will go ahead and remove the space.

Reply:
Great. Those icons were only to show a great representation of the mobile phone in the figma template. You don't need it in your design 😁